Disaster-Response-Platform/backend/Controllers/user_controller.py

```diff
@@ -138,3 +138,37 @@ async def get_user_info(response:Response,username: str, current_user: UserProfi
                         private_account=user.private_account)
 
     return user_info
+
+
+@router.put("/verify")
+def verify_user(response:Response,user: UserUsername, current_user: UserProfile = Depends(authentication_service.get_current_admin_user)):
+    try:
+        verified = authentication_service.verify_user(user.username)
+        response.status_code = HTTPStatus.OK
+        return {f'{user.username} is verified'}    
+    except ValueError as err:
+        err_json = create_json_for_error("Verification error", str(err))
+        response.status_code = HTTPStatus.NOT_FOUND
+        return json.loads(err_json)
+
+@router.put("/unverify")
+def unverify_user(response:Response,user: UserUsername, current_user: UserProfile = Depends(authentication_service.get_current_admin_user)):
+    try:
+        unverified = authentication_service.unverify_user(user.username)
+        response.status_code = HTTPStatus.OK
+        return {f'{user.username} is unverified'}   
+    except ValueError as err:
+        err_json = create_json_for_error("Unverification error", str(err))
+        response.status_code = HTTPStatus.NOT_FOUND
+        return json.loads(err_json)
+
+@router.put("/unauthorize")
+def unverify_user(response:Response,user: UserUsername, current_user: UserProfile = Depends(authentication_service.get_current_admin_user)):
```

Comment:
the function is named identically to the one above, does the code still work?

Reply:
This I will check

Reply:
You are right should be changed. I think @azizamankenova @bgmrsln ?

Reply:
Correction done.